src/mstch.cpp

```diff
@@ -435,7 +435,14 @@ ::mstch::node CXXRecord::methods()
         if (isa<CXXConstructorDecl>(method_decl))
             continue;
         if (method_decl->isOverloadedOperator())
-            continue;
+        {
+            const OverloadedOperatorKind kind
+                = method_decl->getOverloadedOperator();
+            // TODO: Support more operators
+            if (kind != OverloadedOperatorKind::OO_Plus
+                && kind != OverloadedOperatorKind::OO_Star)
+                continue;
```

Comment:
While it is a bit more verbose, I think a `switch` might be clearer as we have more operators.
```suggestion
            switch (kind)
            {
                case OverloadedOperatorKind::OO_Plus:
                case OverloadedOperatorKind::OO_Star:
                     break;  // Allow these kinds.
                default:
                    continue;  // Suppress any other kinds.
            }
```